lib/qesapdeployment.pm

```diff
@@ -295,14 +349,17 @@ sub qesap_prepare_env {
     my $variables = $args{openqa_variables};
     my $provider = $args{provider};
     my %paths = qesap_get_file_paths();
-    my $qesap_conf_src = "sles4sap/qe_sap_deployment/" . $paths{qesap_conf_filename};
 
-    qesap_create_folder_tree();
-    qesap_get_deployment_code();
-    qesap_pip_install();
+    # Option to skip straight to configuration
+    unless ($args{only_configure}) {
```

Comment:
Not for this PR, but I propose to move

```
 qesap_create_folder_tree();
        qesap_get_deployment_code();
        qesap_pip_install();
```

out from this function and leave to the test to call them